relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -193,10 +215,16 @@ fn scrub_resource_identifiers(mut string: &str) -> Option<String> {
         Cow::Owned(scrubbed) => Some(scrubbed),
         Cow::Borrowed(string) => {
             // No IDs scrubbed, but we still want to set something.
-            let url = Url::parse(string).ok()?;
-            let extension = url.path().rsplit_once('.')?.1;
-            let domain = normalize_domain(&url.host()?.to_string(), url.port())?;
-            Some(format!("{domain}/*.{extension}"))
+            // If we managed to parse the URL, we'll try to get an extension.
+            if let Ok(url) = Url::parse(string) {
+                let domain = normalize_domain(&url.host()?.to_string(), url.port())?;
+                // If there is an extension, we add it to the domain.
+                if let Some(extension) = url.path().rsplit_once('.') {
+                    return Some(format!("{domain}/*.{}", extension.1));
+                }
+                return Some(domain);
+            }
+            Some(string.into())
```

Comment:
If we cannot parse a domain, I don't think we want to accept the unfiltered string here, right?
```suggestion
            None
```

Reply:
Since span metrics for resource spans are still protected by a feature flag, I'm actually fine with experimenting here, i.e. leaving `Some(string.into())` and then analyze what data we get.